internal/binutils/disasm_test.go

```diff
@@ -132,6 +132,25 @@ func TestFunctionAssembly(t *testing.T) {
 				{Addr: 0x2001, Text: "instruction two"},
 			},
 		},
+		{
+			plugin.Sym{Name: []string{"_main"}, Start: 0x30000, End: 0x3FFF},
+			`_main:
```

Comment:
nit: here similar to a comment in another test file - a double-quote string with escaped newline would read a bit easier than the backquote string with no indentation on 2+ lines.

Reply:
Done. Exceeding 80 columns in some cases.